app/src/main/java/com/cornellappdev/android/eatery/TimeUtil.java

```diff
@@ -0,0 +1,35 @@
+package com.cornellappdev.android.eatery;
+
+import org.threeten.bp.ZoneId;
+
+public class TimeUtil {
+
+  private static TimeUtil instance;
+
+  public static TimeUtil getInstance() {
+    if (instance == null) {
+      instance = new TimeUtil();
+    }
+
+    return instance;
+  }
+
+  private ZoneId cornellTimeZone;
+
+  public ZoneId getLocalTimeZone() {
+    return ZoneId.systemDefault();
+  }
+
+  public ZoneId getCornellTimeZone() {
+    if (cornellTimeZone == null) {
+      cornellTimeZone = ZoneId.of("America/New_York");
+    }
+
+    if (cornellTimeZone == null) {
+      cornellTimeZone = ZoneId.of("EDT");
+    }
+
```

Comment:
same thing here, just one line the `if` statements and the method can fit in 3 lines

Reply:
I'll change it. I think in general we disagree about vertical space. For me, I use it a little more liberally to (in my opinion) separate logical ideas. I'll try to avoid this in future merges and reduce what I see now!

Reply:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used